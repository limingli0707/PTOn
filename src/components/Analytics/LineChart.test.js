// Link.react.test.js
import React from 'react';
import Line from './LineChart';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

Enzyme.configure({adapter: new Adapter()});


test('Line test', () => {
  const component = shallow(
    <Line/>
  );
//  let profile = component.toJSON();
  expect(toJson(component)).toMatchSnapshot();
});