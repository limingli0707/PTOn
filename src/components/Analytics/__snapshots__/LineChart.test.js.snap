// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line test 1`] = `
<div
  className="App"
>
  <Chart
    chartActions={null}
    chartEvents={null}
    chartType="LineChart"
    chartWrapperParams={Object {}}
    className=""
    columns={null}
    controls={null}
    data={
      Array [
        Array [
          "Date",
          "Total Views",
          "Total Sales",
        ],
        Array [
          "8/7",
          4000,
          1500,
        ],
        Array [
          "13/7",
          5000,
          2200,
        ],
        Array [
          "14/7",
          5000,
          2800,
        ],
        Array [
          "15/7",
          4500,
          2200,
        ],
        Array [
          "17/7",
          6500,
          3300,
        ],
        Array [
          "17/7",
          5500,
          2200,
        ],
      ]
    }
    diffdata={null}
    formatters={null}
    getChartEditor={null}
    getChartWrapper={[Function]}
    graphID={null}
    graph_id={null}
    height="100%"
    legendToggle={false}
    legend_toggle={false}
    options={
      Object {
        "intervals": Object {
          "style": "line",
        },
        "legend": "none",
        "lineWidth": 3,
        "title": "",
      }
    }
    render={null}
    rootProps={Object {}}
    rows={null}
    spreadSheetQueryParameters={
      Object {
        "gid": 1,
        "headers": 1,
      }
    }
    spreadSheetUrl={null}
    style={Object {}}
    toolbarID={null}
    toolbarItems={null}
    width="100%"
  />
</div>
`;
